tailscale/provider.go

```diff
@@ -113,6 +124,9 @@ func providerConfigure(_ context.Context, d *schema.ResourceData) (interface{},
 	}
 
 	userAgent := d.Get("user_agent").(string)
+	if userAgent == "" {
+		userAgent = provider.UserAgent("terraform-provider-tailscale", providerVersion)
```

Comment:
This is essentially an identical call to what we were making before, but at this point the underlying `provider.TerraformVersion` value used in the [UserAgent call](https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/schema/provider.go#L497) is populated properly.